src/style/avatar.css

```diff
@@ -0,0 +1,34 @@
+@import url("color.css");
+
+.avatar {
+    border-radius: 50%;
+}
+
+.avatar-letter {
+    font-weight: 500;
+    text-align: center;
+    vertical-align: middle;
+    background-color: var(--charcoal-black);
+    color: var(--charcoal-white);
+}
+
+.avatar-large {
+    width: 75px;
+    height: 75px;
+    font-size: 3rem;
+    line-height: 4.5rem;
+}
+
+.avatar-regular {
+    height: 50px;
+    width: 50px;
+    font-size: 2rem;
+    line-height: 3rem;
+}
+
+.avatar-small {
+    height: 25px;
+    width: 25px;
+    font-size: 1rem;
+    line-height: 1.5rem;
+}
```

Comment:
Add rems instead of px for height and width of avatars

Reply:
@romabulani value will not change for avatar with screen size change therefor i have used px instead of rem.

Reply:
@romabulani avatar size is fixed inrespective of screen size therefore used px instead of rem.

Reply:
Yes, as rems is based on font size, it will be good with responsiveness. but you want to keep it fixed, then its fine.